frontend/apps/crates/entry/module/card-quiz/edit/src/router.rs

```diff
@@ -28,24 +28,11 @@ pub fn render(state: Rc<Router>) {
         dominator::routing::url()
             .signal_ref(|url| Route::from_url(url))
             .for_each(clone!(state => move |route| {
-                match route {
-                    Route::Module(route) => {
-                        match route {
-                            ModuleRoute::Edit(kind, jig_id, module_id) => {
-                                match kind {
-                                    ModuleKind::CardQuiz => {
-                                        let app = create_state(jig_id, module_id);
-                                        render_page_body(app.clone());
-                                        *state.app.borrow_mut() = Some(app);
-                                    }
-                                    _ => {}
-                                }
-                            }
-                            _ => {}
-                        }
-                    },
-                    _ => {}
-                };
+                if let Route::Module(ModuleRoute::Edit(ModuleKind::CardQuiz, jig_id, module_id)) = route {
```

Comment:
I just think that this line reads easier

"If the `Module` we are `Edit`ing is a `CardQuiz`, then"